projects/fsastorefrontstyles/scss/components/cms/_find-agent-search-list.scss

```diff
@@ -42,4 +42,11 @@
   cx-pagination {
     display: inline-flex;
   }
+  %cx-store-finder-map {
+    .cx-store-map {
+      @include media-breakpoint-down(md) {
```

Comment:
I encountered on the one more problem. Map doesn't appear on the tablet screens, so I overrode their media-breakpoint and added medium breakpoint instead of small one.